### PR TITLE
Change usage rules for the Named Info registry

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

**Review by nedmsmith — APPROVED** [review]
LGTM

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
Is it the wrong one, though?

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -290,15 +289,10 @@ namespace under which the value must be understood.
...
-to the "Named Information Hash Algorithm Registry" {{-named-info}}.
-When it is carried as `text`, there are no requirements with regards to its
-format.  In general, the `int` encoding is RECOMMENDED.  The `text` encoding
-should only be used when the `digest` type conveys reference value
-measurements that are matched verbatim with Evidence that uses the same
-convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
+A digest represents the value of a hashing operation together with the hash algorithm used.
+The type of the digest algorithm identifier can be either `int` or `text` and is interpreted according to the {{-named-info}} registry.
+Specifically, `int` values are matched against "ID" entries, `text` values are matched against "Hash Name String" entries.
+The `int` encoding is RECOMMENDED.
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:295]:
```suggestion
Whenever possible, using the `int` encoding is RECOMMENDED.
```

**Review by nedmsmith — APPROVED** [review]
LGTM

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**